### DISCUSSION TITLE
Shelly 2.5 stopped showing power monitoring

### DISCUSSION THREAD

**Discussion by blade3609** [arendst/Tasmota#18140] [Q&A] [answered]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
Shelly 2.5 stopped showing power monitoring , only showing temeprature

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [v ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ v] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [v ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ v] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ v] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ v] Device used (e.g., Sonoff Basic): _Shelly 2.5____
- [v ] Tasmota binary firmware version number used: __12.4 normal and sensors___
  - [ v] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
stat/dshelly1/RESULT = {"Module":{"0":"Shelly 2.5"}}
10:38:28.487 MQT: stat/dshelly1/RESULT = {"GPIO0":{"320":"Led_i1"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"0":"None"},"GPIO4":{"224":"Relay1"},"GPIO5":{"193":"Switch_n2"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"640":"I2C SDA"},"GPIO13":{"192":"Switch_n1"},"GPIO14":{"608":"I2C SCL"},"GPIO15":{"225":"Relay2"},"GPIO16":{"3456":"ADE7953 IRQ1"},"GPIO17":{"4736":"ADC Temp"}}

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
{"Status":{"Module":0,"DeviceName":"dshelly1","FriendlyName":["Tasmota",""],"Topic":"dshelly1","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0,"StatusRetain":0}}
10:38:57.970 MQT: stat/dshelly1/STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota-sensors.bin.gz","RestartReason":"External System","Uptime":"0T00:07:59","StartupUTC":"2023-03-09T14:30:58","Sleep":50,"CfgHolder":4617,"BootCount":9,"BCResetTime":"2023-03-08T01:21:25","SaveCount":38,"SaveAddress":"F9000"}}
10:38:58.000 MQT: stat/dshelly1/STATUS2 = {"StatusFWR":{"Version":"12.4.0(sensors)","BuildDateTime":"2023-02-24T15:32:29","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"373/699"}}
10:38:58.021 MQT: stat/dshelly1/STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["C&M",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A192800000000","00000081","00006000","00004000","00000000"]}}
10:38:58.060 MQT: stat/dshelly1/STATUS4 = {"StatusMEM":{"ProgramSize":698,"Free":304,"Heap":17,"ProgramFlashSize":1024,"FlashSize":2048,"FlashChipId":"1540EF","FlashFrequency":40,"FlashMode":"DOUT","Features":["00000809","8F9A8787","0415A005","B7F7BFCF","05DA9BC4","64367CC7","00084052","20000000","54000020","0000C081"],"Drivers":"1,2,3,4,5,6,7,8,9,10,12,14,16,17,20,21,24,29,34,62,65,66","Sensors":"1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,17,18,19,20,21,22,26,31,34,37,39,40,42,43,45,51,52,55,56,58,59,62,64,66,67,74,98,103"}}
10:38:58.105 MQT: stat/dshelly1/STATUS5 = {"StatusNET":{"Hostname":"dshelly1-7196","IPAddress":"192.168.1.11","Gateway":"192.168.1.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.1.1","DNSServer2":"0.0.0.0","Mac":"68:C6:3A:FB:9C:1C","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}}
10:38:58.126 MQT: stat/dshelly1/STATUS6 = {"StatusMQT":{"MqttHost":"107.170.25.27","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_FB9C1C","MqttUser":"cris","MqttCount":1,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
10:38:58.151 MQT: stat/dshelly1/STATUS7 = {"StatusTIM":{"UTC":"2023-03-09T14:38:58","Local":"2023-03-09T10:38:58","StartDST":"2023-03-26T02:00:00","EndDST":"2023-10-29T03:00:00","Timezone":"-04:00","Sunrise":"06:46","Sunset":"18:44"}}
10:38:58.171 MQT: stat/dshelly1/STATUS10 = {"StatusSNS":{"Time":"2023-03-09T10:38:58","Switch1":"OFF","Switch2":"OFF","ANALOG":{"Temperature":56.7},"TempUnit":"C"}}
10:38:58.185 MQT: stat/dshelly1/STATUS11 = {"StatusSTS":{"Time":"2023-03-09T10:38:58","Uptime":"0T00:08:00","UptimeSec":480,"Heap":17,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":22,"MqttCount":1,"POWER1":"OFF","POWER2":"OFF","Wifi":{"AP":1,"SSId":"C&M","BSSId":"14:59:C0:42:F3:91","Channel":8,"Mode":"11n","RSSI":78,"Signal":-61,"LinkCount":1,"Downtime":"0T00:00:04"}}}

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
The monitoring info just disappeared

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
I need to see power monitoring info, but i only see temperature

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._
![image](https://user-images.githubusercontent.com/25647415/224058550-03c1c978-6a69-4bed-a74a-aefaff4e6bb3.png)

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
I did a reset 5, power cycle, re-configured and still noting, im on the latest version. I even tried and installed the sensors tasmota bin, still nothing

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by barbudor** [comment]
What is the result of command `backlog i2cdrivers; i2cscan`
You should have driver number 7 enabled (without a `!` mark)

**Comment by blade3609** [comment]
this is the result

backlog i2cdrivers; i2cscan
10:54:11.581 MQT: stat/dshelly1/RESULT = {"Command":"Unknown"}
10:54:11.809 MQT: stat/dshelly1/RESULT = {"I2CScan":"No devices found"}

does that mean that the power monitoring chip died?

**Reply by sfromis** [reply]
> does that mean that the power monitoring chip died?

That's quite plausible. At least Tasmota finding no devices on the bus (if correctly configured) indicates no answer from the sensor.

**Comment by blade3609** [comment]
I2cDriver
10:56:00.517 MQT: stat/dshelly1/RESULT = {"I2CDriver":"7,8,9,10,11,12,13,14,15,17,18,20,24,29,31,36,41,42,44,46,48,69,76"}

**Comment by vic42** [comment]
I'd (physically) power cycle the device.

**Comment by blade3609** [comment, marked as answer]
i did that already, a couple of times, removed power completely to the device for about an hour.

**Reply by barbudor** [reply]
As your template seems OK with I2C SDA on Gpio12 and SCL on gpio14, I light up a candle for the monitoring chip
Requiescat in pace

**Reply by blade3609** [reply]
for some strange reason it started working again lol

backlog i2cdrivers; i2cscan
stat/dshelly1/RESULT = {"I2CDriver":"7,8,9,10,11,12,13,14,15,17,18,20,24,29,31,36,41,42,44,46,48,69,76"}
stat/dshelly1/RESULT = {"I2CScan":"Device(s) found at 0x38"}
